Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -58,96 +58,31 @@ extension Store {
   }
 }
 
-extension ViewStore {
```

Comment:
Ah I think the main issue is this should be wrapped in `#if compiler(>=5.4)` and we should keep the deprecation notices around.

Just curious, but what's preventing you from using Xcode 12.5?

Reply:
Ah sorry 🤦, thank you. Let me update the PR.

About the question, there are just some other priorities in the project. We'll be upgrading to 12.5.1 soon.

Reply:
Sorry @stephencelis . I had couple copy of the TCA source code and I messed with git clients. Let me close that PR and open a fresh one.